### ISSUE TITLE
The VideoCaptureDevice class does not trigger the event new frame on windows 7 OS though it works like a charm in windows 10

### ISSUE THREAD

**Issue by mithunvs** [andrewkirillov/AForge.NET#32]

**Comment by andrewkirillov** [comment]
Just tested - works without issues on my Windows 7. It might be system/camera specific issue, so I would suggest debugging with source code on your side.

**Comment by djhuds2k** [comment]
Did you ever resolve this issue? Have exactly the same issue on one Windows 7 Pro system. It works fine on lots of other systems with Win 7 but for some reason this one isn't working. it's not a driver issue as video capture works fine in other apps.

**Comment by andrewkirillov** [comment]
No. Was no able to reproduce on my system. Try taking the source code and debugging.

**Comment by selway** [comment]
> No. Was no able to reproduce on my system. Try taking the source code and debugging.

I got the same problem.  When I debugging with source code, I found that it was because  `ISampleGrabber` stopped calling back `VideoCpatureDevice.Grabber.BufferCB`.  But I never found out what caused it.

**Comment by GholibjonMadiyarov** [comment]
У меня также некоторое время работала 32-разрядная версия Windows 7, но она внезапно перестала работать.

**Comment by GholibjonMadiyarov** [comment]
Когда я вернул Windows в прежнее состояние, она снова запустилась, но через несколько дней снова перестала работать.
